### PR TITLE
added ImageCompressHelper;

### PR DESCRIPTION
**Pull request by vorobevamv**
added webp to ImageFormats

### CODE REVIEW

**Review by VLiamin — APPROVED** [review]

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      return Task.Run(() =>
+      {
+        try
+        {
+          string compressedContent = inputBase64;
+          int quality = 90;
+
+          while (Convert.FromBase64String(compressedContent).Length / 1000 > MaxWeighInKB)
+          {
+            if (extension == svg)
```
Comment (ndoriomedov) [inline src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: Maybe use switch case?

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            {
+              byte[] byteString = Convert.FromBase64String(inputBase64);
+              using MemoryStream ms = new MemoryStream(byteString);
+              Bitmap bmp = SvgDocument.Open<SvgDocument>(ms).Draw();
+              ImageConverter converter = new ImageConverter();
+              byteString = (byte[])converter.ConvertTo(bmp, typeof(byte[]));
+              compressedContent = Convert.ToBase64String(byteString);
+              extension = png;
```
Comment (ndoriomedov) [inline src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: Maybe move to separate method?

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+              Image image = Image.Load(Convert.FromBase64String(compressedContent), out IImageFormat imageFormat);
+              compressedContent = image.ToBase64String(imageFormat).Split(',')[1];
+              quality -= 10;
+
+              if (quality == 0)
+              {
+                break;
+              }
+            }
```
Comment (ndoriomedov) [inline src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: And this?

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            else
+            {
+              Image image = Image.Load(Convert.FromBase64String(inputBase64));
+              image.Mutate(x => x.BackgroundColor(SixLabors.ImageSharp.Color.White));
+              compressedContent = image.ToBase64String(JpegFormat.Instance).Split(',')[1];
+              extension = jpeg;
```
Comment (ndoriomedov) [inline src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: Same

**Review by ndoriomedov — APPROVED** [review]

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs (hunk 1 above)
Comment (vorobevamv) [reply src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: done

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs (hunk 2 above)
Comment (vorobevamv) [reply src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: done

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs (hunk 3 above)
Comment (vorobevamv) [reply src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: done

File: src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs (hunk 4 above)
Comment (vorobevamv) [reply src/Kernel.ImageSupport/Helpers/ImageCompressHelper.cs]: done

**Pull request merged**